examples/graphene_apps/python_worker/README.md

```diff
@@ -166,6 +168,16 @@ https://creativecommons.org/licenses/by/4.0/
 
 - Knows issues are captured in GitHub : https://github.com/hyperledger/avalon/issues/621
 
+## FAQs
+
+  1. What to do when I see an error `insmod: ERROR: could not insert module gsgx.ko: File exists` while installing Graphene SGX driver?
+
+     Ignore the error and proceed. You have probably already installed the Graphene SGX driver.
+
+  2. What should I do if I see an error `ModuleNotFoundError: No module named 'docker'`?
+
+     You need to install python paackage for docker using `python3 -m pip install docker`.
+
```

Comment:
The FAQ looks out of scope for this ReadMe.md. Also we need to remove gsgx driver and use in kernel driver with FSGSBASE patch soon.

Reply:
build_gsc_python_worker.sh script does throw such an error. The gsc.py from Graphene call for this dependency.